app/frontend/src/components/FileInput.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <v-file-input
+      counter
+      label="Upload your file"
+      prepend-icon="mdi-paperclip"
+      show-size
+      v-model="files"
+    />
+    <v-textarea
+      auto-grow
+      hint="JSON format for key-value pairs"
+      label="Contexts"
+      :mandatory="true"
+      name="contexts-input"
```

Comment:
Don't need the name field on these since they are v-model bound if I'm not mistaken (never need to refer back to them in a form post, model binding is used instead)

Reply:
Removed